### ISSUE TITLE
Genre metadata set to "Blues" on MP3 export

### ISSUE THREAD

**Issue by Spekular** [LMMS/lmms#5014]
Exporting a track to MP3 appears to set the genre metadata to "Blues".

Reported by Hayden/Pyrif on the LMMS Discord:

> question: why do all of my LMMS rendered files come with a "Blues" genre tag by default? not sure if it does this when rendering to OGG or WAV, but definitely does when rendering to MP3.

& by Twitter user @nimi_stix

> Another bug!
> LMMS v1.2.0 rc-8 export to MP3 has tag error. all songs' genres display Blues

I've reproduced this on 1.2 RC8, checking the tag with Mp3tag. Hayden checked with Audacity.

**Comment by PhysSong** [comment]
It coincides with the fact that Blues corresponds to genre ID 0. I searched for the source code of LAME and found a fix.
```diff
diff --git a/src/core/audio/AudioFileMP3.cpp b/src/core/audio/AudioFileMP3.cpp
index cce7ec8e4..a00cdf611 100644
--- a/src/core/audio/AudioFileMP3.cpp
+++ b/src/core/audio/AudioFileMP3.cpp
@@ -120,6 +120,7 @@ bool AudioFileMP3::initEncoder()
 	lame_set_brate(m_lame, bitRate);
 
 	// Add a comment
+	id3tag_init(m_lame);
 	id3tag_set_comment(m_lame, "Created with LMMS");
 
 	return lame_init_params(m_lame) != -1;
```
`id3tag_init` initializes the genre metadata to 0xFF which means undefined.

**Comment by PhysSong** [comment]
Fixed via 47d6388182e9105c0cf7e87d494dd2db593350c5.
